src/KM_Game.pas

```diff
@@ -968,6 +967,14 @@ function TKMGame.IsMapEditor: Boolean;
 end;
 
 
+function TKMGame.IsMPGameSpeedUpAllowed: Boolean;
+begin
+  Result := (fGameMode in [gmMulti, gmMultiSpectate])
+        and (fNetworking.NetPlayers.GetConnectedCount = 1)
+        and not fWaitingForNetwork;
```

Comment:
It would be better to check that all other human players have "Dropped" set in KM_NetPlayers rather than relying on fWaitingForNetworking during a disconnection/reconnection. Maybe add a function to KM_NetPlayers called OnlyOnePlayerRemaining (or a better name).